extensions/Redis/Redis/RedisMemory.cs

```diff
@@ -85,7 +85,7 @@ public async Task CreateIndexAsync(string index, int vectorSize, CancellationTok
     public async Task<IEnumerable<string>> GetIndexesAsync(CancellationToken cancellationToken = default)
     {
         var result = await this._search._ListAsync().ConfigureAwait(false);
-        return result.Select(x => (string)x!);
+        return result.Select(x => ((string)x!)).Where(x => x.StartsWith($"{this._config.AppPrefix}-", StringComparison.Ordinal)).Select(x => x.Substring(this._config.AppPrefix.Length + 1));
```

Comment:
thoughts about making `-` part of the prefix? 

e.g. instead of 

> AppPrefix = `km` and `x.StartsWith($"{this._config.AppPrefix}-")`

setting

> AppPrefix = `km-` and then `x.StartsWith(this._config.AppPrefix)`

(plus changing other places accordingly)

Reply:
Done, my only thing would be that without the separator being outside the prefix you'll have odd looking keys/indexes. E.g. `kmtests` if the user doesn't add the prefix themselves. I'm probably overthinking as they'd never need to actually look at the keys unless they were troubleshooting something directly in Redis.

Reply:
you could change the default to "km-" in `RedisConfig` ctor, similarly to the Postgres connector

Reply:
done.